<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDefined</class>
 <widget class="QWidget" name="UserDefined">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resonrces.qrc">
    <normaloff>:/images/app.ico</normaloff>:/images/app.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	color:#fff;
	/*background-color: rgb(240,240,240);*/
	background-color: rgb(24,28,40);
}
QLabel{
	background-color: rgb(41,45,57);
}
QCheckBox{
	background-color: rgb(41,45,57);
}

QPushButton{
	color:white;
	border:2px solid #147e3a;
	background-color:#019b48;
	width:50px;
	height:30px;
}

QPushButton:hover{
	background-color:#019b48;
	font-size:12px;
	font-weight:bold;
}

QLabel{
	color:white;
}

QGroupBox{
	border:2px solid rgb(41,45,57);
	color:white;
	background-color: rgb(41,45,57);
	border-radius:4px;
}

QLineEdit{	
	background-color: rgb(255, 255, 255);
	border:1px solid #147e3a;
	width:100px;
	height:30px;
	color:black;
}
QComboBox{
	background-color: rgb(255, 255, 255);
	width:100px;
	height:27px;
	border-radius:4px;
	border:1px solid rgb(41,45,57);
	color:black;
}

QLineEdit:hover{
	border:1px solid rgb(0,149,255);
}

QTabWidget:pane{
	border-top:1px solid rgb(240, 240, 240);
	/*border:1px solid rgb(41,45,57);*/
}

QTabBar::tab{
	color:white;
	width:50px;
	height:25px;
	background-color:#019b48;
	margin-right: 5px;
	margin-bottom:-6px;
}

QTabBar::tab:selected{
	background-color:rgb(0,149,255);
	font-weight:bold;
	height:28px;
	border:1.5px solid rgb(0,110,255);
}

QMessageBox{
	background-color:rgb(41,45,57);
}

QMessageBox QPushButton:enabled{
	border-radius: 5px;
	width: 50px;
	height: 20px;
	background-color: #019b48;
	color: white;
	font-size: 12px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,4,1">
   <item>
    <widget class="QGroupBox" name="positionLoopGroupBox_3">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_20" stretch="1,4">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_21">
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>开关量参数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_11">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="gridGroupBox_8">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border:1px solid rgb(41,45,57);
	height:25px;
	width:100px;
	border-radius:5px;
}
QLineEdit:hover{
	border:1px solid rgb(0,149,255);
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9" columnstretch="1,1,1,1,1,1,1,1">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>37</number>
         </property>
         <property name="verticalSpacing">
          <number>3</number>
         </property>
         <item row="1" column="5">
          <widget class="QCheckBox" name="checkBox_13">
           <property name="text">
            <string>13</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBox_9">
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QCheckBox" name="checkBox_6">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="checkBox_10">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QCheckBox" name="checkBox_14">
           <property name="text">
            <string>14</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QCheckBox" name="checkBox_11">
           <property name="text">
            <string>11</string>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QCheckBox" name="checkBox_15">
           <property name="text">
            <string>15</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBox_1">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QCheckBox" name="checkBox_3">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QCheckBox" name="checkBox_5">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QCheckBox" name="checkBox_12">
           <property name="text">
            <string>12</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_8">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QCheckBox" name="checkBox_4">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QCheckBox" name="checkBox_7">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBox_16">
           <property name="text">
            <string>16</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="checkBox_17">
           <property name="text">
            <string>17</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="checkBox_18">
           <property name="text">
            <string>18</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QCheckBox" name="checkBox_19">
           <property name="text">
            <string>19</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QCheckBox" name="checkBox_20">
           <property name="text">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QCheckBox" name="checkBox_21">
           <property name="text">
            <string>21</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QCheckBox" name="checkBox_22">
           <property name="text">
            <string>22</string>
           </property>
          </widget>
         </item>
         <item row="2" column="7">
          <widget class="QCheckBox" name="checkBox_23">
           <property name="text">
            <string>23</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_24">
           <property name="text">
            <string>24</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="checkBox_25">
           <property name="text">
            <string>25</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="checkBox_26">
           <property name="text">
            <string>26</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QCheckBox" name="checkBox_27">
           <property name="text">
            <string>27</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QCheckBox" name="checkBox_28">
           <property name="text">
            <string>28</string>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QCheckBox" name="checkBox_29">
           <property name="text">
            <string>29</string>
           </property>
          </widget>
         </item>
         <item row="3" column="6">
          <widget class="QCheckBox" name="checkBox_30">
           <property name="text">
            <string>30</string>
           </property>
          </widget>
         </item>
         <item row="3" column="7">
          <widget class="QCheckBox" name="checkBox_31">
           <property name="text">
            <string>31</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_hall">
           <property name="text">
            <string>霍尔方向</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="positionLoopGroupBox_4">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_22" stretch="1,4">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_23">
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>寄存器参数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_12">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="gridGroupBox_9">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border:1px solid rgb(41,45,57);
	height:25px;
	width:100px;
	border-radius:5px;
}
QLineEdit:hover{
	border:1px solid rgb(0,149,255);
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_10" columnstretch="2,1,1,1,1,1">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>47</number>
         </property>
         <property name="verticalSpacing">
          <number>3</number>
         </property>
         <item row="3" column="5">
          <widget class="QLineEdit" name="lineEdit_19"/>
         </item>
         <item row="5" column="3">
          <widget class="QLineEdit" name="lineEdit_13"/>
         </item>
         <item row="6" column="3">
          <widget class="QLineEdit" name="lineEdit_14"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>霍尔角度偏移（度）</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>参数19</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>参数10</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QLineEdit" name="lineEdit_18"/>
         </item>
         <item row="4" column="5">
          <widget class="QLineEdit" name="lineEdit_20"/>
         </item>
         <item row="1" column="5">
          <widget class="QLineEdit" name="lineEdit_17"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>力传感器滤波系数（2位精度）</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="lineEdit_7"/>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="lineEdit_11"/>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="lineEdit_10"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>开环匹配电流Iq（1位精度）</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>力传感器零点</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>参数8</string>
           </property>
          </widget>
         </item>
         <item row="8" column="5">
          <widget class="QLineEdit" name="lineEdit_24"/>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>参数13</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>参数17</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="lineEdit_9"/>
         </item>
         <item row="6" column="4">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>参数22</string>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QLineEdit" name="lineEdit_15"/>
         </item>
         <item row="7" column="4">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>参数23</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>参数12</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item row="5" column="4">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>参数21</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="lineEdit_8"/>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>参数14</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>版本修改时间</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>参数11</string>
           </property>
          </widget>
         </item>
         <item row="8" column="4">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>参数24</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>参数9</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>力传感器系数（4位精度）</string>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QLineEdit" name="lineEdit_21"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>PT滤波系数（2位精度）</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>参数18</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item row="6" column="5">
          <widget class="QLineEdit" name="lineEdit_22"/>
         </item>
         <item row="7" column="5">
          <widget class="QLineEdit" name="lineEdit_23"/>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>参数15</string>
           </property>
          </widget>
         </item>
         <item row="8" column="3">
          <widget class="QLineEdit" name="lineEdit_16"/>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>参数16</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="lineEdit_12"/>
         </item>
         <item row="4" column="4">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>参数20</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="positionLoopGroupBox_5">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_24" stretch="4">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QGroupBox" name="gridGroupBox_10">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border:1px solid rgb(41,45,57);
	height:25px;
	width:100px;
	border-radius:5px;
}
QLineEdit:hover{
	border:1px solid rgb(0,149,255);
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_11" columnstretch="2,1,2">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>37</number>
         </property>
         <property name="verticalSpacing">
          <number>3</number>
         </property>
         <item row="0" column="1">
          <widget class="QPushButton" name="sureBtn">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resonrces.qrc"/>
 </resources>
 <connections/>
</ui>
